### PR TITLE
Mes 7585 non pass finalisation back button

### PR DESCRIPTION
**Pull request by solomonsimukoko**
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/app/pages/non-pass-finalisation/non-pass-finalisation.page.ts [hunk 1]
```diff
@@ -96,6 +96,7 @@ export class NonPassFinalisationPage extends PracticeableBasePageComponent imple
   testCategory: CategoryCode;
 
   constructor(
+    private navController: NavController,
```
Comment (matthew2564) [inline src/app/pages/non-pass-finalisation/non-pass-finalisation.page.ts]: This is unused so can we remove it please

File: src/app/pages/non-pass-finalisation/__tests__/non-pass-finalisation.page.spec.ts [hunk 2]
```diff
@@ -213,5 +219,11 @@ describe('NonPassFinalisationPage', () => {
           .toHaveBeenCalledWith(NonPassFinalisationValidationError('notRequiredControl is blank'));
       }));
     });
+    fdescribe('navigateToDebrief', () => {
```
Comment (matthew2564) [inline src/app/pages/non-pass-finalisation/__tests__/non-pass-finalisation.page.spec.ts]: Can we please remove the f prefix

File: ios/App/App.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (matthew2564) [inline ios/App/App.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Please remove this file from PR

File: capacitor.config.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (matthew2564) [inline capacitor.config.json]: Please undo

**Comment by andrewsetterfield** [comment]
Does the back that this is a back button with a navigation attached cause any issues? every other instance i can see in the code has covereted the back button to a standard button.

**Review by matthew2564 — APPROVED** [review]

**Review by andrewsetterfield — APPROVED** [review]

**Pull request merged**